test-app/lib/ember-blog/addon/routes/new.js

```diff
@@ -0,0 +1,19 @@
+import { inject as service } from '@ember/service';
+import Route from '@ember/routing/route';
+
+let count = 0;
```

Comment:
@aklkv curious why did you want to store the state in such a way? this might be counterintuitive

Reply:
I am open to suggestions but this was the easiest way to save the state between reloads